deployments/data-hub/data-hub--staging.yaml

```diff
@@ -0,0 +1,230 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: data-hub--staging
+  namespace: data-hub
+  annotations:
+    fluxcd.io/automated: 'true'
+
+spec:
+  releaseName: data-hub--staging
+  chart:
+    repository: https://airflow-helm.github.io/charts
+    name: airflow
+    version: 8.4.1
+
+  values:
+    ingress:
+      enabled: true
+      web:
+        host: data-hub--flux--staging.elifesciences.org
+        annotations:
+          nginx.ingress.kubernetes.io/auth-url: "https://oauth-proxy.elifesciences.org/oauth2/auth"
+          nginx.ingress.kubernetes.io/auth-signin: "https://oauth-proxy.elifesciences.org/oauth2/start?rd=https%3A%2F%2F$host$request_uri"
+      flower:
+        host: data-hub--flux--staging.elifesciences.org
+        path: "/flower"
+        annotations:
+          nginx.ingress.kubernetes.io/auth-url: "https://oauth-proxy.elifesciences.org/oauth2/auth"
+          nginx.ingress.kubernetes.io/auth-signin: "https://oauth-proxy.elifesciences.org/oauth2/start?rd=https%3A%2F%2F$host$request_uri"
+    airflow:
+      legacyCommands:
+        enabled: true
+      config:
+        AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION: "True"
+        AIRFLOW__CORE__DAG_CONCURRENCY: "100"
+        AIRFLOW__CORE__MAX_ACTIVE_RUNS_PER_DAG: "25"
+        AIRFLOW__CELERY__WORKER_CONCURRENCY: "21"
+        AIRFLOW__CORE__REMOTE_LOGGING: "True"
+        AIRFLOW__CORE__REMOTE_BASE_LOG_FOLDER: "s3://staging-elife-data-pipeline/airflow-logs"
+        AIRFLOW__CORE__REMOTE_LOG_CONN_ID: "aws_default"
+        AIRFLOW__KUBERNETES_ENVIRONMENT_VARIABLES__AIRFLOW__CORE__REMOTE_LOGGING: "True"
+        AIRFLOW__KUBERNETES_ENVIRONMENT_VARIABLES__AIRFLOW__CORE__REMOTE_BASE_LOG_FOLDER: "s3://staging-elife-data-pipeline/airflow-logs"
+        AIRFLOW__CELERY__FLOWER_URL_PREFIX: "/flower"
+      image:
+        repository: elifesciences/data-hub-with-dags_unstable
+        tag: f1d91c7b417ca7c338a1ef4463035d49e99b52f9
```

Comment:
Another future improvement would be to update this container to follow the tagging convention `<branchname>-<shortsha>-<date>.<timestamp>` that we use in others, e.g. `master-1234567-20220111.1234`

Reply:
Agreed. I'm also a little unsure as to whether the entirely separate image is required, as opposed to correctly labeled images. All this felt a little out of scope for just now though.